packages/fluentui/accessibility/src/behaviors/Table/gridRowBehavior.ts

```diff
@@ -15,4 +15,6 @@ export default gridRowBehavior;
 export type GridRowBehaviorProps = {
   /** Indicates if a table row is header. */
   header?: boolean;
+  /** Indicated is table row is selected.  */
+  selected?: boolean;
```

Comment:
Should be this added to specification?

Reply:
It's already in gridRowNestedBehavior specification, and since it doesn't affect gridRowHeaderBehavior, I'd leave it only in gridRowNestedBehavior spec, if you agree with that? :)